tests/application/cms/test_views.py

```diff
@@ -995,3 +949,139 @@ class TestChartBuilder(_TestVisualisationBuilder):
 
 class TestTableBuilder(_TestVisualisationBuilder):
     ROUTE_NAME = "cms.create_table"
+
+
+class TestRemoveDataSourceView:
+    def setup(self):
+        self.saved_config = {**current_app.config}
```

Comment:
Do we need these? I removed them from my view tests as they didn't seem to do anything?

Reply:
They do, if you have tests that change the app config.